### ISSUE TITLE
Bug in Sensor.TYPE_GAME_ROTATION_VECTOR

### ISSUE THREAD

**Issue by jucallej** [bq/aquaris-M5#2]
Hi BQ team!

I have detected a bug in the Sensor.TYPE_GAME_ROTATION_VECTOR. 

Using the Sensor.TYPE_ROTATION_VECTOR
(http://developer.android.com/reference/android/hardware/SensorEvent.html), it provides four correct values on onSensorChanged. But using Sensor.TYPE_GAME_ROTATION_VECTOR, that according to the documentation should provide the same values, but without using the magnetometer, the fourth value is always 0, when it shouldn't.

For example in:

@Override
public void onSensorChanged(SensorEvent event) {
    if (event.accuracy != SensorManager.SENSOR_STATUS_UNRELIABLE && event.sensor.getType() == Sensor.TYPE_GAME_ROTATION_VECTOR){

```
    SensorManager.getRotationMatrixFromVector(mRotationMatrixFromVector, event.values);
    Log.d(LOG_TAG, "values[0]: " + event.values[0] + ", values[1]: " + event.values[1] + ", values[2]: " + event.values[2] + ", values[3]: " + event.values[3]);
```

}

In this case value[3] returns always 0, although it should change when moving the device. The device is using, Android 5.0.2. In settings I can see I'm using kernel: 3.10.49, number of compilation: 1.2.4_20151021-0942, in case it is useful.

If you need more information, please let me know.

I already contacted BQ support who redirected me to opensource [at] bq.com, but I haven't had an answer in several weeks. Could you solve this issue.

Thanks and best regards,
Julian

**Comment by Kra1o5** [comment]
Hello Julian,

Thank you for contacting us & sorry for the delay.

The latest version available for Aquaris M5 is 3.2.2 (Android 5.1.1). 

If you have troubles with OTA update please contact with the support team at: http://www.bq.com/uk/support/aquaris-m5 .

If problem persist after update your phone, contact us again and we will be happy to give you a solution as soon as possible.

We are at your disposal
Alejandro Lopez

**Comment by jucallej** [comment]
Hi Kra1o5,

I hadn't updated the device since I reported this by mail. Updating the device solved the issue. I'm closing this thread.

Thanks and best regards,
Julian
